frontend/src/components/PageController.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { RootSchema } from "../store/root-schema";
+import LoginPage from "./LoginPage";
+import LobbyPage from "./LobbyPage";
+import { connect } from "react-redux";
+
+interface Props {
+  showLogin: boolean;
+}
+
+const PageController: React.FunctionComponent<Props> = ({ showLogin }: Props) => {
+  return (
+    <>
+      {showLogin && <LoginPage />}
+      {!showLogin && <LobbyPage />}
+    </>
+  );
+};
+
+const mapStateToProps = (state: RootSchema) => ({
+  showLogin: state.socketState === "closed" || state.socketState === "connecting",
```

Comment:
I don't think we should base current view on a socket state, e.g. Lobby and Game will have the same socket state. Simple enum in store will be fine